src/main/java/org/cristalise/kernel/persistency/outcomebuilder/field/StringField.java

```diff
@@ -1,410 +1,415 @@
-/**

```

Comment:
Please make sure your editor does not change the line delimiter (unix/windows line ending) of the file.